src/test/java/uk/gov/cshr/controller/signup/SignupControllerTest.java

```diff
@@ -241,8 +271,8 @@ public void shouldReturnSignupIfInviteAuthorised() throws Exception {
         mockMvc.perform(
                 get("/signup/" + code)
                         .with(CsrfRequestPostProcessor.csrf()))
-                .andExpect(status().isOk())
-                .andExpect(view().name("signup"));
+                .andExpect(status().is3xxRedirection())
+                .andExpect(redirectedUrl("/signup/request"));
```

Comment:
The last expectation and the unit test method name are not in synch. The existing method seems to be correct.

Reply:
Method shouldReturnSignupIfInviteAuthorised above is test only to check functionality if code is authorised or not. The setting of invite to valid is part of "when" and it does not test the validity of the code. The code is set to valid to reach to the code which check the code is authorised or not. So test name should reflect only if invite is authorised or not.

Reply:
DONE. Code updated.